components/docs-chef-io/content/automate/servicenow_integration_reference.md

```diff
@@ -145,6 +145,38 @@ The **Field Maps** establishes a relationship between a field in an import set t
 
 {{< figure src="/images/automate/snow_integration_transform_map_scripts1.png" alt="Transform Map Scripts">}}
 
+## Discovery
+
+[ServiceNow](https://www.servicenow.com/) Discovery feature finds applications and devices on your network, and then updates the CMDB with the information it finds. It discovers both physical and logical components, including virtual machines, servers, storage, databases, applications, and more.
+
+Refer [Discovery](https://docs.servicenow.com/bundle/paris-it-operations-management/page/product/discovery/reference/r-discovery.html) page for information on Discovery and its types.
+
+### Identification and Reconciliation
+
+Reconciliation is the synchronization of two or more matching database segments to ensure consistency across them.
+
+Identification rules identify new CIs and existing CIs. It applies to a CI class, and it can be single or multiple with different priorities.
+
+Reconciliation rules specify which discovery sources can update a table or a set of table attributes, and the precedence order among these discovery sources. You can define these rules at the parent or at the child class level. Without reconciliation rules, discovery sources are overwritten with each other's updates to attribute values.
+
+The Identification and Reconciliation rules maintain the integrity of the CMDB by managing duplicate CIs and controlling updates to CIs when multiple data sources are used to create and update CI records. These rules help prevent duplication of CI records, reconcile CI attributes, reclassify CIs, and allow authoritative data sources to update CI records in the CMDB.
```

Comment:
You might put this paragraph at the top of this section so the user gets an overview of Identification and Reconciliation before they get an explanation of what each does.

Reply:
sure